package.json

```diff
@@ -38,6 +38,9 @@
     "eslint-plugin-standard": "3.0.1",
     "pg-copy-streams": "0.3.0"
   },
+  "peerDependencies": {
+    "bl": "^2"
```

Comment:
Sorry I'm kinda bad w/ `peerDependencies` versus `optionalDependencies` and when to use each.  Does making `bl` a `peerDependency` mean it'll be installed along w/ the main `pg` module?  Or just that if the user installs it on their end it needs to be `bl@2.x`?

Reply:
Yes it's a bit confusing as they kind of work backwards to how you'd expect from the naming.

Peer dependencies aren't installed by default as its expected the "main" app's package.json will include them. I think npm prints a warning at npm install time if there are unmet peer dependencies.

Optional dependencies are installed by default if there's a version available for your target platform. The primary usage is if you've got a platform specific module (say with an arch specific DLL) and you only want it installed if it exists. For pure-JS modules that don't restrict the arch I think they're always installed as they'd always match up.

It's a bit of a hack to use peerDependencies as true optional dependencies as it's a warning and not an error if they're not actually installed.

Thinking about this some more, maybe the simplest solution is not to include any dependency item at all and just add a note to the docs about it. Anybody using this would have to install "bl" at the top level anyway to use it in their app and the require statement within pg will be able to find it.